.evg.yml

```diff
@@ -95,7 +98,98 @@ tasks:
     commands:
       - func: "static code analysis"
 
+  - name: sbom
+    commands:
+      - func: "generate sbom"
+      - func: "add team name to sbom"
```

Comment:
I think you can merge this func with the previous one. It doesn't have to be so granular.
I also don't know how I feel with using func versus having a list of commands being part of a function.
It makes the func longer, but at the same time, having all piece be under the same function means that there it is clear that they have to happen in that order and that what the previous function produces is expected to exist for the next function.

Reply:
Originally, it was one function; however, it would stop executing after `./gradlew cyclonedxBom` for some reason, so I made two separate functions. However, I can combine the two and just have two `- command: shell.exec` in one function.